tokens_testnet_example.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Suggestion: put the token configs into a directory, perhaps `cfg` in the root of the repo.